### PR TITLE
Remove `--no-location` argument to rxgettext

### PR DESCRIPTION
**Pull request by smcclellan**
The `--no-location` argument means that the POT file doesn't include file
names or file lines. This was originally added to reduce churn in CI from extra
commits when the POT file changed.

Now that we have an `update_pot` rake task, only actual changes to the strings
will result in a new file being generated. This should solve the original
problem of CI churn, meaning we can add the file (and line numbers) back in,
helping translators see where strings come from.

Fixes https://tickets.puppetlabs.com/browse/INTL-22

### CODE REVIEW

**Comment by Magisus** [comment]
I feel like maybe we should get rid of that "prefer single quoted string when possible" rule in rubocop. What do you think?

**Comment by smcclellan** [comment]
I was just going to fix all of those instances, working on it now.

**Comment by highb** [comment]
Boo, [single quotes suck](https://www.viget.com/articles/just-use-double-quoted-ruby-strings)!

**Comment by Magisus** [comment]
This works as advertised, and interacts correctly with the `update_pot` task: if a string's line number changes but no strings themselves have changed, the POT file does not get updated.

File: lib/gettext-setup/gettext_setup.rb [hunk 1]
```diff
@@ -40,31 +41,31 @@ def self.initialize(locales_path, options = {})
...
   end
 
-  def self.add_repository_to_chain(project_name,options)
+  def self.add_repository_to_chain(project_name, options)
     repository = FastGettext::TranslationRepository.build(project_name,
-                                                          :path => locales_path,
-                                                          :type => options[:file_format] || :po,
-                                                          :ignore_fuzzy => false)
-    @@translation_repositories[project_name] = repository unless @@translation_repositories.key? project_name
+                                                          path: locales_path,
```
Comment (Magisus) [inline lib/gettext-setup/gettext_setup.rb]: +1 to updating this hash syntax as well

**Comment by Magisus** [comment]
LGTM 👍

**Comment by highb** [comment]
Yay! When will the gem release happen?

**Comment by smcclellan** [comment]
@highb Just completed: https://rubygems.org/gems/gettext-setup/versions/0.19

**Pull request merged**